nntrainer/include/layer.h

```diff
@@ -36,8 +36,10 @@ namespace nntrainer {
  * @brief     Enumeration of cost(loss) function type
  *            0. MSR ( Mean Squared Roots )
  *            1. ENTROPY ( Cross Entropy )
- *            2. ENTROPY_SIGMOID (Cross Entropy amalgamated with sigmoid for stability)
- *            2. ENTROPY_SOFTMAX (Cross Entropy amalgamated with softmax for stability)
+ *            2. ENTROPY_SIGMOID (Cross Entropy amalgamated with sigmoid for
+ * stability)
+ *            2. ENTROPY_SOFTMAX (Cross Entropy amalgamated with softmax for
+ * stability)
  *            2. Unknown
```

Comment:
Numbers? is it related with this PR?

Reply:
Not really related though, it seems it need fixing. I changed numbering :) Thanks.